access-token-management/src/AccessTokenManagement.OpenIdConnect/AuthenticateResultCache.cs

```diff
@@ -1,13 +1,20 @@
-﻿// Copyright (c) Duende Software. All rights reserved.
+// Copyright (c) Duende Software. All rights reserved.
+// See LICENSE in the project root for license information.
+
+/* Unmerged change from project 'AccessTokenManagement.OpenIdConnect(net9.0)'
+Before:
 // Licensed under the Apache License, Version 2.0. See LICENSE in the project root for license information.
+After:
+// See LICENSE in the project root for license information.
+*/
+// See LICENSE in the project root for license information.
```

Comment:
Needs fixing.

Reply:
yeah.. I saw those and thought they were introduced by a previous merge. No idea that dotnet format added these. They have been fixed now.